views/scss/qti/_choice.scss

```diff
@@ -45,38 +52,18 @@
                 [data-eliminable="container"] {
                     left: $elimListStyleCbOffsetLeft * 1px;
                 }
-                // li
-                .qti-choice {
-                    // when checked
-                    &.eliminated {
-                        [data-eliminable="container"] {
-                            padding-left: $elimListStyleCbOffsetLeft * 1px;
-                            //&::before {
-                            //    left: ($elimListStyleCbOffsetLeft - $captionOffsetLeft) * 1px !important;
-                            //}
-                        }
-                    }
-                    &:first-child {
-                        &.eliminated {
-                            [data-eliminable="container"] {
-                                // label above ol
-                                &::before {
-                                    left: ($elimListStyleCbOffsetLeft - $captionOffsetLeft) * 1px !important;
-                                }
-                            }
-                        }
-                    }
-                }
             }
-        }
-        //both label and checkbox
-        [data-eliminable] {
-            display:block;
+            //both label and checkbox
+            [data-eliminable] {
+                display: block;
+            }
         }
         // eliminator outer layer
         .qti-choice {
             // when checked
             &.eliminated {
+                background: white(.7);
```

Comment:
**nitpick:** Please use variables for the colors. There is a list available here: https://github.com/oat-sa/tao-core-ui-fe/blob/master/scss/inc/_colors.scss
This is important for theming.